[2025-10-07T10:17:00.601+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:17:00.702+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=26 [queued]>
[2025-10-07T10:17:00.759+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=26 [queued]>
[2025-10-07T10:17:00.778+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:17:00.865+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:01:57.519057+00:00
[2025-10-07T10:17:00.901+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=519) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:17:00.909+0000] {standard_task_runner.py:72} INFO - Started process 717 to run task
[2025-10-07T10:17:00.909+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:01:57.519057+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp65vvckyl', '--map-index', '26']
[2025-10-07T10:17:00.920+0000] {standard_task_runner.py:105} INFO - Job 41: Subtask process_batch
[2025-10-07T10:17:00.995+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:17:01.163+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=26 [running]> on host e3278e0f0bbc
[2025-10-07T10:17:02.056+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:01:57.519057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:01:57.519057+00:00'
[2025-10-07T10:17:02.059+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:17:02.073+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DTG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.077+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DTE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.079+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DTD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.080+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DTC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.085+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DTB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.093+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DTA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.095+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DT4: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.096+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.098+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DST: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.099+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.111+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.111+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.112+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.116+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.116+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.117+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DSC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.122+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DS3: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.123+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DRL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.124+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DRI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.124+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DRH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.125+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DRG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.127+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DRC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.128+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DQC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.133+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.134+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.136+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.137+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.137+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.138+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.138+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.138+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DPC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.139+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DP3: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.141+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DP2: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.146+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DP1: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.147+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DOP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.147+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DOC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.148+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.150+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.152+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.159+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.160+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.160+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.161+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.161+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.161+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DND: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.162+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.162+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.163+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DNA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.163+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DMS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.164+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DMN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:02.165+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:17:02.197+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:17:02.201+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:01:57.519057+00:00, map_index=26, execution_date=20251007T100157, start_date=20251007T101700, end_date=20251007T101702
[2025-10-07T10:17:02.577+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:17:02.630+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T10:17:02.702+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
